app/models/organisation.rb

```diff
@@ -0,0 +1,20 @@
+require 'active_model'
+
+class Organisation
+  include ActiveModel::Model
+
+  attr_accessor(
+    :title,
+    :content_id,
+    :link,
+    :slug,
+    :organisation_type,
+    :organisation_state,
+    :document_count,
+    :acronym
```

Comment:
Does it make sense to only be pulling out the attributes we know we need for now? We can always expand this later if we find out we need more. I.e: I don't think we need document_count, acronym, organisation type, possibly not slug either.

Reply:
I was thinking about this, I do agree that is easy to add them in later. I think I will remove what we don't need. I will keep slug. 

I think we need the the document count to make sure that the order of the organisations displayed are correct - but then again this list that comes back is in order of most tagged content....

Reply:
Sounds good. 

It may be better to explicitly sort rather than rely on rummager returning these in the right order, in case that somehow changes?

Reply:
Yeah that was what I was thinking. I think that can be done in a presenter.